pkg/operator/options/options.go

```diff
@@ -64,9 +64,26 @@ type Options struct {
 	setFlags map[string]bool
 }
 
-func (o *Options) AddFlags(fs *flag.FlagSet) {
+type FlagSet struct {
+	*flag.FlagSet
+}
+
+// BoolVarWithEnv defines a bool flag with a specified name, default value, usage string, and fallback environment
+// variable.
+func (fs *FlagSet) BoolVarWithEnv(p *bool, name string, val bool, envVar string, usage string) {
```

Comment:
```suggestion
func (fs *FlagSet) BoolVarWithEnv(p *bool, name string, envVar string, val bool, usage string) {
```

IMO, this should be the argument ordering

Reply:
We probably should add testing, the reason I didn't is the interesting part to test is the merge semantic with the env var IMO and without a refactor to enable mocking it out idk what a good methodology to test that would be. I didn't think a refactor of that would really be in scope here but maybe it should be?